math/maxima/Portfile

```diff
@@ -39,7 +39,7 @@ subport maxima-devel {
     # Date:  Sat Oct 29 16:24:21 2022 +0000
     # commit 5b42cccfd9eee54d0458d8f4e63e6b44c55d48a1
     version     5.46-dev-20221029
-    revision    0
+    revision    1
```

Comment:
are you sure that the only port which should be revbump is maxima?

Reply:
To be frank, I don't understand why Maxima needs to be bumped at all. It's extremely rare for Common Lisp software to depend on a specific version of SBCL, unless they are just loading FASLs directly into SBCL. In that case, yes we should revbump the port. In any case, I think Maxima is the only software that potentially needs revbump.